thirdeye/thirdeye-pinot/src/test/resources/schema/create-schema.sql

```diff
@@ -27,7 +27,7 @@ create table if not exists anomaly_merge_config_index (
     name varchar(200) not null,
     base_id bigint(20) not null,
     create_time timestamp,
-    update_time timestamp default current_timestamp,
+    update_time timestamp default current_timestamp on update current_timestamp,
```

Comment:
Did this work? There was some issue with this. Not sure if local test db support them